__tests__/cache/SimpleCache.spec.js

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { createStore } from 'redux';
+import SimpleCache from '../../src/cache/SimpleCache';
+import * as actions from '../../src/cache/actions';
+
+const fakeError = new Error('Fake Error');
+const fakeData = {
+  status: 200,
+  ok: true,
+  body: {
+    fakeData: true,
+  },
+};
+
+const createScenario = (dispatch, actionKeys, hash) => {
+  const actionOrder = actionKeys.map((key) => actions[key]);
+  actionOrder.forEach((actionCreator) => {
+    dispatch(
+      actionCreator({
+        hash,
+        error: fakeError,
+        value: fakeData,
+      })
+    );
+  });
+};
+
+describe('SimpleCache', () => {
```

Comment:
These tests are so similar to the ones for `ImmutableCache` That you may want to consider creating some kind of test harness for these and future caches

Reply:
Yeah in hindsight I should rig up something similar to how I test `useFetchye` with all the providers.

Reply:
Agreed, as the caches follow an adapter pattern ensuring that every cache conforms to an api will be important. Would also be nice to see the required cache api documented.

Reply:
Fixed in https://github.com/americanexpress/fetchye/pull/8/commits/b7a24b8c48335a0bec2bc15e611d2af45dd4defa with `testCacheInterface`. Swapped to regular snapshots given non specificity of test harness.